src/blas/backends/cublas/cublas_scope_handle.hpp

```diff
@@ -23,27 +23,6 @@
 #else
 #include <CL/sycl.hpp>
 #endif
-#if __has_include(<sycl/context.hpp>)
-#if __SYCL_COMPILER_VERSION <= 20220930
-#include <sycl/backend/cuda.hpp>
-#endif
-#include <sycl/context.hpp>
-#else
-#include <CL/sycl/backend/cuda.hpp>
-#include <CL/sycl/context.hpp>
-#endif
-
-// After Plugin Interface removal in DPC++ ur.hpp is the new include
-#if __has_include(<sycl/detail/ur.hpp>)
-#include <sycl/detail/ur.hpp>
-#ifndef ONEMKL_PI_INTERFACE_REMOVED
-#define ONEMKL_PI_INTERFACE_REMOVED
-#endif
-#elif __has_include(<sycl/detail/pi.hpp>)
-#include <sycl/detail/pi.hpp>
-#else
-#include <CL/sycl/detail/pi.hpp>
-#endif
```

Comment:
The header file `cublas_handle.hpp` below can be completely removed now.

Reply:
`cublas_handle` is still [being used](https://github.com/oneapi-src/oneMKL/blob/8f4312ef966420b9b8b4b82b9d5c22e2c91a1fe7/src/blas/backends/cublas/cublas_scope_handle_hipsycl.cpp#L27) by the hipSYCL version, I'm not sure what to do about that.

Reply:
Right, there are some known issues with AdaptiveCpp and not enough workforce to fix them so let's just keep the file.

Reply:
I think `cublas_handle.hpp` doesn't need to be included in `cublas_scope_handle.hpp` though?